### ISSUE TITLE
Question about the bool_masked_pos for pre-training

### ISSUE THREAD

**Issue by Sean-C71** [ZiyangZhang0511/Echo-Vison-FM#2]
Thanks for sharing this excellent work.
However, I'm kinda confused by the pretrain.py at row 124.

`bool_masked_pos = (random_tensor > args.mask_ratio)`

Suppose mask_ratio is 0.85
**This will make 85% as 0, which should strands for unmasked patches, right? 
However, it should be 85% masked patches.**

According to the modeling_videomae.py from the huggingface transformers/models/videomae source code at line 623:
```python
@can_return_tuple
@auto_docstring
def forward(
    self,
    pixel_values: torch.FloatTensor,
    bool_masked_pos: torch.BoolTensor,
    head_mask: Optional[torch.Tensor] = None,
    **kwargs: Unpack[TransformersKwargs],
) -> VideoMAEForPreTrainingOutput:
    r"""
    bool_masked_pos (`torch.BoolTensor` of shape `(batch_size, sequence_length)`):
        Boolean masked positions. Indicates which patches are masked (1) and which aren't (0). Each video in the
        batch must have the same number of masked patches. Sequence length is `(num_frames // tubelet_size) *
        (image_size // patch_size) ** 2`.
```

I'm not sure If my understanding is correct.
Do I miss something?
